src/Skoruba.IdentityServer4.Admin/Program.cs

```diff
@@ -61,7 +61,9 @@ private static IConfiguration GetConfiguration(string[] args)
             var configurationBuilder = new ConfigurationBuilder()
                 .SetBasePath(Directory.GetCurrentDirectory())
                 .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
-                .AddJsonFile("serilog.json", optional: true, reloadOnChange: true);
+                .AddJsonFile($"appsettings.{environment}.json", optional: true, reloadOnChange: true)
```

Comment:
Do we really need this line ? I think apps already works with appsettings{environemnt}.json by default.

Reply:
Imho yes, see https://github.com/skoruba/IdentityServer4.Admin/pull/551/files#r397750808